oxd-gen-client/src/test/java/io/swagger/client/api/GetOpDiscoveryConfigTest.java

```diff
@@ -0,0 +1,23 @@
+package io.swagger.client.api;
+
+import io.swagger.client.model.GetOpDiscoveryConfigParams;
+import io.swagger.client.model.GetOpDiscoveryConfigResponse;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import static org.testng.AssertJUnit.assertNotNull;
+
+public class GetOpDiscoveryConfigTest {
+    @Parameters({"opHost", "opDiscoveryPath"})
+    @Test
+    public void test(String opHost, String opDiscoveryPath) throws Exception {
+            DevelopersApi api = Tester.api();
```

Comment:
reformat this class with Intellij IDEA. It somehow has 8 spaces instead of standard 4. Other classes looks good.

Reply:
Changes done.